docs/cli/create-api-impl.md

```diff
@@ -7,14 +7,18 @@
 
 #### Syntax
 
-`ern create-api-impl <apiName>`  
+`ern create-api-impl <apiName> [apiImplName]`  
 
 **Arguments**
 
 `<apiName>`
 
 * The package descriptor of the API for which to create an API implementation project
 
+`[apiImplName]`
+* The name of the API IMPL project.  API IMPL name must contain only alphabets.
```

Comment:
Here and in the whole page please replace `API IMPL` by `API implementation`.

Reply:
`API Impl name must contain only alphabets` => `The API implementation name must only contain upper and/or lower case letters`.

As discussed, might create at some point an entry (not part of this PR) in the glossary for `Electrode Native module name` (tbd) and clarify it there.